src/product-card.css

```diff
@@ -0,0 +1,82 @@
+.product-card {
+  box-sizing: border-box;
+  justify-content: center;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.product-card__description {
+  padding: 0 16px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-card__code {
+  padding: 4px 16px 0;
```

Comment:
set padding 16px to .product-card so you wont need horizontal margins and paddings on inner elements